python/kwiver/vital/tests/test_camera_intrinsics.py

```diff
@@ -51,8 +50,8 @@ def test_full_init(self):
         CameraIntrinsics(10, (5, 5), 1.2, 3.1, [4, 5, 6])
 
     def test_get_focal_length(self):
-        ntools.assert_equal(CameraIntrinsics().focal_length, 1.)
-        ntools.assert_equal(CameraIntrinsics(5.2).focal_length, 5.2)
+        assert(CameraIntrinsics().focal_length == 1.)
```

Comment:
Nitpick: `Pytest` has `assertEqual`: https://docs.python.org/3/library/unittest.html

Not a huge deal to use `assert` here and everywhere else, but it might be cleaner. In general, anything that `Nose` had, `Pytest` has, but in camelCase instead of snake_case

Reply:
You're right, pytest is able to drive and support most of those other testing frameworks, however pytest itself does not support those features. This test was intended to be an example of  the pure pytest functionality, while the rest of the testing suite served as an example of pytest running other testing frameworks like unittest and nose so we can get a sense which direction to take. Additionally, the tool mentioned above that can be used to automate the transition from nose to pytest performs the refactor in the same way done here, so if we wish to retain the unittest based tests, we'll need to change our approach.